10_installation/10_install_server/snippets/apt_key.txt.snippet

```diff
@@ -0,0 +1,16 @@
+We will first import our key into apt using the following command:
+
+----
+
+wget --quiet -O- "https://www.rudder-project.org/apt-repos/rudder_apt_key.pub" | sudo apt-key add -
+
+----
+
+Our key fingerprint is:
```

Comment:
Can you add here something like "Please check that the fingerprint matches the one displayed by this command". ?